runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsConfig.java

```diff
@@ -60,4 +61,15 @@ default boolean enableRestorationBars()
 	{
 		return true;
 	}
-}
+
+	@ConfigItem(
+		keyName = "hideAfterCombatDelay",
+		name = "Hide after combat delay",
+		description = "Time required to hide status bars after no longer in combat."
+	)
+	@Units(Units.SECONDS)
```

Comment:
Game ticks would be a better unit of measurement since that's the unit of time this change actually operates on. That would also let you drop the `Instant.now()` duration comparisons in favor of using `client.getTickCount()`.